ppr-ui/tests/setup.ts

```diff
@@ -0,0 +1,70 @@
+// // setup.ts
+import { afterEach, beforeAll, beforeEach, vi } from 'vitest'
+import { config } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import vuetify from '@/plugins/vuetify'
+import * as matchers from 'vitest-axe/matchers'
+import 'vitest-axe/extend-expect'
+import { expect } from 'vitest'
+
+// Extend vitest with axe matchers
+expect.extend(matchers)
+
+// Define Pinia/Vuetify/router globally
+const pinia = createPinia()
+setActivePinia(createPinia())
+
+// Add properties to the wrapper
+config.global.plugins.push([vuetify, pinia])
+// Suppress Vue warnings
+config.global.config.warnHandler = () => null
+global.css = { supports: () => false }
+
+beforeAll(() => {
+  // Mock the entire vue-pdf-embed module
+  vi.mock('vue-pdf-embed', () => {
+    // Replace the component with a dummy component or return an empty object
+    return {
+      default: {
+        render () {
+          return null
+        }
+      }
+    }
+  })
+
+  // Mock the WysiwygEditor component
+  vi.mock('@/components/common/WysiwygEditor.vue', () => {
```

Comment:
Is this just an example/placeholder or we need a mocked editor before all tests?

Reply:
Thats correct, this is a mocked editor. Will probably refine this to mock just the imported package (we shouldn't need to unit test imported code) or mock the functions that cause the breakage in the unit test env.